Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp

```diff
@@ -288,10 +288,6 @@ void AuxiliaryProcessProxy::didFinishLaunching(ProcessLauncher*, IPC::Connection
     if (!connectionIdentifier)
         return;
 
-#if PLATFORM(MAC) && USE(RUNNINGBOARD)
-    m_lifetimeAssertion = ProcessAssertion::create(xpc_connection_get_pid(connectionIdentifier.xpcConnection.get()), "Lifetime assertion"_s, ProcessAssertionType::Foreground);
```

Comment:
Without this, what prevents suspension of background tabs?

Reply:
We talked about this offline but I'll post here for others. `WebKit:Suspended` does not actually suspend on either macOS or iOS and `ProcessThrottler` will always hold at least a `WebKit:Suspended` assertion. This is confusing so I'll change this patch to make sure we always hold a "Background" assertion so that it's more readable and less surprising.

Reply:
I added this assertion back behind an internal runtime flag while we wait for a more suitable assertion class from runningboard. I think it would be useful to be able to test this in its current form and gather data while we iterate on a final design.